market/management/commands/create_customers.py

```diff
@@ -0,0 +1,29 @@
+from django.core.management.base import BaseCommand
+from market.models import Customers
+import random
+from datetime import datetime, timedelta, date
+
+
+class Command(BaseCommand):
+    help = 'Create n customers'
+
+    def add_arguments(self, parser):
+        parser.add_argument('-n', type=int, help='count of customers', default=10, required=False, )
+
+    def handle(self, *args, **kwargs):
+        n = kwargs['n']
+        for i in range(int(n)):
```

Comment:
В парсере уже указан тип int
```suggestion
        for i in range(n):
```